Off-Top-App/off-top-java
# OT-5 | Create MySQL Database

### What I Did:
- Created a MySQL databased called offTop
- Created Models that scaffolded into MySQL tables
- Installed and setup dependencies required to create entities and for scaffolding spring code into MySQL

### How To Test My Branch:
- **Prereq (Assuming you have Spring-Boot running already)**: Download MySQL
    1. Be able to use `mysql -u root -p`
- Before you run the Java Spring project you will need to create the database locally since it isn't hosted yet. To do so you will need to: 
   1.  access mysql shell by running `mysql -u root -p` and then entering all needed credentials
   2. Once you're in MySQL shell you will need to run `CREATE DATABASE offTop;` 
   3. To see if it was created properly you will need to run `USE offTop;` if it says Empty Set then you can proceed.
- After creating a local instance of the database, you can now run Spring-Boot project. If all is well you can run `SHOW tables;` to see this below.
<img width="649" alt="Screen Shot 2020-02-12 at 10 09 47 PM" src="https://user-images.githubusercontent.com/26909882/74406416-74761780-4de4-11ea-8ba0-dc32d66d3107.png">
- If you want to see what is in the tables you can run `describe <table-name>;`. So if you want to see the users tables you can run `describe users;`
